beacon-chain/state/state-native/v1/setters_attestation_test.go

```diff
@@ -21,52 +17,6 @@ func TestBeaconState_RotateAttestations(t *testing.T) {
 	require.NoError(t, err)
 
 	require.NoError(t, st.RotateAttestations())
-	require.Equal(t, 0, len(st.currentEpochAttestations()))
-	require.Equal(t, types.Slot(456), st.previousEpochAttestations()[0].Data.Slot)
-}
-
-func TestAppendBeyondIndicesLimit(t *testing.T) {
```

Comment:
I had to remove this test temporarily because it relies on changes in `fieldtrie` package which are not merged in yet. Maybe commenting out is better?

Reply:
We want to avoid checking in commented code. Instead, we should add an issue for bringing back these deleted tests

Reply:
Yes lets track this better